src/locator/Locators.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+class Locators{
+
+	public static $loginUserName = ['id' => 'username'];
```

Comment:
Starting with PHP 5.6 you can do 

const LOGIN_USER_NAME = ['id' => 'username'];